Assets/MRTK/Examples/Demos/UX/Slider/Scripts/ShowSliderValue.cs

```diff
@@ -25,7 +25,7 @@ public void OnSliderUpdated(SliderEventData eventData)
 
             if (textMesh != null)
             {
-                textMesh.text = $"{eventData.NewValue:F2}";
+                textMesh.text = $"{eventData.NewValue*90.0f:F2}";
```

Comment:
any reason for this change?

Reply:
Same as below; I believe this was a relic of targeting dev rather than the march branch

Reply:
Sorry, this is my temporary change to use the slider to test out hand drop (camera facing threshold) value. I'll remove before opening PR.